libguard/guard_exception.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <exception>
+#include <sstream>
+#include <string>
+
+namespace openpower
+{
+namespace guard
+{
+namespace exception
+{
+
+class GuardException : public std::exception
+{
+  public:
+    explicit GuardException(const std::string& message) : message(message){};
+
+    virtual const char* what() const noexcept override
```

Comment:
Why you are using both virtual and override? The `what()` is virtual function in `std::exception` class.